fabio/ext/meson.build

```diff
@@ -0,0 +1,49 @@
+py = import('python').find_installation()
+dep_py = py.dependency()
```

Comment:
You already have this defined from the top-level meson.build, so you do not need to re-import it.

Reply:
Thanks, I a complete newbe with meson